packages/flutter/lib/src/widgets/form.dart

```diff
@@ -455,9 +455,9 @@ class FormField<T> extends StatefulWidget {
     this.validator,
     this.initialValue,
     this.enabled = true,
-    AutovalidateMode? autovalidateMode,
+    AutovalidateMode? autoValidateMode,
     this.restorationId,
-  }) : autovalidateMode = autovalidateMode ?? AutovalidateMode.disabled;
+  }) : autoValidateMode = autoValidateMode ?? AutovalidateMode.disabled;
```

Comment:
Similar to #160268, it'd be best to have a deprecation period rather than a hard breaking change here:

```dart
FormField({
  @Deprecated(...)
  AutovalidateMode? autovalidateMode,
  AutovalidateMode? autoValidateMode,
}) : autoValidateMode = autoValidateMode ?? autovalidateMode ?? AutovalidateMode.disabled;
```